### DISCUSSION TITLE
v0.0.4 Alpha

### DISCUSSION THREAD

**Discussion by LuanRoger** [LuanRoger/xbox_launcher#54] [Announcements]
### New Home Page layout
![home](https://user-images.githubusercontent.com/46284513/217535378-310be29f-aca5-4d0e-bb92-c19d41b003e5.png)
__Made in https://github.com/LuanRoger/xbox_launcher/pull/53__

* System menu on Home Page, accessible pressing BACK on controller or F1 on keyboard.
* Configuration button on top of Home Page.
* Fix shortcuts issues in https://github.com/LuanRoger/xbox_launcher/pull/51
* Fix game sorting in https://github.com/LuanRoger/xbox_launcher/pull/52
* Fix volume percentage in https://github.com/LuanRoger/xbox_launcher/pull/52

**Full Changelog**: https://github.com/LuanRoger/xbox_launcher/compare/v0.0.3-alpha...v0.0.4-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/LuanRoger/xbox_launcher/releases/tag/v0.0.4-alpha'>v0.0.4 Alpha</a>.</em>

**Comment by whattheheo132** [comment]
how do you exit the launcher?
